README.md

```diff
@@ -196,6 +205,9 @@ hide = true
 > [!TIP]
 > To run a script with spaces in the launch command, you will need to use `\\` before any space characters, e.g. `launch_command = "/home/user/GOG\\ Games/Stardew\\ Valley/start.sh"`. Other options can be passed in directly without escaping spaces.
 
+> [!INFO]
+> Note that `launch_env` only takes effect on games that are invoked directly by the launch command (defined either in the config file or discovered). For discovered games, only itch.io games are currently supported. All other discovered games are invoked indirectly and the custom environment variables may not be respected in those cases.
```

Comment:
I think this can go too. I'll have a think on how to let people know it might not work how they expect depending on the launcher.